features/networking/multus.feature

```diff
@@ -1348,3 +1348,48 @@ Feature: Multus-CNI related scenarios
       | ["spec"]["containers"][0]["name"]                          | macvlan-bridge-whereabouts          |
     Then the step should succeed
     And the pod named "macvlan-bridge-whereabouts-pod1" becomes ready
+    
+  # @author weliang@redhat.com
+  # @case_id OCP-29742
+  @admin
+  Scenario: Log pod IP and pod UUID when pod start	
+    Given the multus is enabled on the cluster
+    And I store all worker nodes to the :nodes clipboard
+    # Create the net-attach-def via cluster admin
+    Given I have a project
+    When I run oc create as admin over "<%= BushSlicer::HOME %>/testdata/networking/multus-cni/NetworkAttachmentDefinitions/whereabouts-macvlan.yaml" replacing paths:
```

Comment:
@weliang1 pls use the `Given I obtain...` step to get the testdata.  Check out #1227 for examples

Reply:
@pruan-rht update to use the way as you suggested. thanks!